packages/firestore/src/remote/serializer.ts

```diff
@@ -644,6 +634,12 @@ export function toMutation(
     return fail('Unknown mutation type ' + mutation.type);
   }
 
+  if (mutation.fieldTransforms && mutation.fieldTransforms.length > 0) {
```

Comment:
`mutation.fieldTransforms` is never false.

Reply:
Removed.